### PR TITLE
Adding article about <details> for advent calendar 2024

### PR DESCRIPTION
**Pull request by jpedroribeiro**

### CODE REVIEW

File: hell/adventcalendar/2024/23/index.md [hunk 1]
```diff
@@ -2,51 +2,188 @@
 title: "The devil is in the &lt;details&gt;"
```
Comment (jpedroribeiro) [inline hell/adventcalendar/2024/23/index.md:2]: @matuzo this renders fine on the page but on the browser title bar it display as text. Not sure if there's anything that can be done. If not, I'd advise in removing the `<...>` for readability? It's up to you.

**Comment by cssence** [comment]
Hey @jpedroribeiro, great article. If you are up for it, how about you end it with the details' `name` attribute, which gives us true accordions with pure HTML:
https://developer.mozilla.org/en-US/blog/html-details-exclusive-accordions/

(Browser support looks good already: https://caniuse.com/mdn-html_elements_details_name)

/cc @matuzo

**Comment by jpedroribeiro** [comment]
@cssence great suggestion, added. Also gave me a chance to at least include Chrome latest info on styling those elements. 
Thanks for the feedback!

**Comment by matuzo** [comment]
Hey J. Pedro,

thanks a lot for your great post. We've added some comments directly in your file in the advent24 branch of the HTMHell repository. Sorry that we're not using GitHub and its comment feature as intended but our approach makes it easier for us since not all authors are using GitHub.

Please review our comments, make your changes if necessary, and submit another PR. Your post goes online in 28 days. It would be great if you could submit your changes a couple of days earlier. If you have questions please ask them here.

Thanks a lot,
Manuel, Saptak, and Karl

**Pull request merged**